include/zephyr/dt-bindings/clock/stm32_common_clocks.h

```diff
@@ -20,35 +20,34 @@
 /** Clock divider */
 #define STM32_CLOCK_DIV(div)	(((div) - 1) << STM32_CLOCK_DIV_SHIFT)
 
-/** STM32 MCO configuration values */
-#define STM32_MCO_CFGR_REG_MASK     0xFFFFU
-#define STM32_MCO_CFGR_REG_SHIFT    0U
-#define STM32_MCO_CFGR_SHIFT_MASK   0x3FU
-#define STM32_MCO_CFGR_SHIFT_SHIFT  16U
-#define STM32_MCO_CFGR_MASK_MASK    0x1FU
-#define STM32_MCO_CFGR_MASK_SHIFT   22U
-#define STM32_MCO_CFGR_VAL_MASK     0x1FU
-#define STM32_MCO_CFGR_VAL_SHIFT    27U
+/** Helper macros to pack RCC clock source selection register info in the DT */
+#define STM32_DT_RCC_CLKSRC_REG_MASK	0xFFFFU
+#define STM32_DT_RCC_CLKSRC_REG_SHIFT	0U
+#define STM32_DT_RCC_CLKSRC_SHIFT_MASK	0x3FU
+#define STM32_DT_RCC_CLKSRC_SHIFT_SHIFT	16U
+#define STM32_DT_RCC_CLKSRC_MASK_MASK	0x1FU
+#define STM32_DT_RCC_CLKSRC_MASK_SHIFT	22U
+#define STM32_DT_RCC_CLKSRC_VAL_MASK	0x1FU
+#define STM32_DT_RCC_CLKSRC_VAL_SHIFT	27U
```

Comment:
Since the scope of these macros is now quite large (MCO and others), it would make sense to make the name more generic.
So replace `CLKSRC` by `CLK` or other generic proposal (could be `REG` but then we'd have `REG_REG` occurences).

Reply:
I can replace `RCC_CLKSRC` with `CLKSEL` since these are expected to be used for clock selection configuration: `STM32_DT_CLKSEL_xxx`.

Reply:
> I can replace `RCC_CLKSRC` with `CLKSEL` since these are expected to be used for clock selection configuration: `STM32_DT_CLKSEL_xxx`.

Do you propose this also applies to `STM32_DT_RCC_CLKSRC_SELECT` also ?